server/controllers/index.js

```diff
@@ -0,0 +1,101 @@
+const Questions = require('../database/schema');
+
+// Create a new question
+exports.create = (req, res) => {
```

Comment:
You could potentially use the same function for create/update if you use something like findByIdAndUpdate in mongoose. If you set the upsert option to true, it will create a new record if one with the given id isn't found, otherwise it will update the document that matches the id
 https://mongoosejs.com/docs/api.html#model_Model.findByIdAndUpdate

Reply:
Thanks for reviewing and the suggestion.

While I initially tried that out, I actually chose this route in the meantime due to wanting to create/update based on the productId (i.e. Numbers 1-100), rather than an ObjectId mongoose creates for us.